docs/consumer-configuration.md

```diff
@@ -0,0 +1,89 @@
+- **java.security.krb5.conf**: location of the krb5.conf file for the Kerberos server.
+- **sasl.kerberos.service.name**: Kafka Kerberos service name. The default value is _kafka_.
+- **jaas.enabled**: establishes whether JAAS is enabled or disabled. The default value is _NO_.
+- **java.security.auth.login.config**: location of the jaas.conf file for Kafka Kerberos.
+- **sasl.mechanism**: configures the SASL mechanism to use to connect to the Kafka cluster. The default value is _GSSAPI_.
+- **value.subject.name.strategy**: default value name strategy. The default value is _io.confluent.kafka.serializers.subject.TopicRecordNameStrategy_.
+- **key.subject.name.strategy**: default key name strategy. The default value is _io.confluent.kafka.serializers.subject.TopicRecordNameStrategy_.
+- **ssl.enabled**: establishes whether SSL is enabled or disabled. The default value is _NO_.
+- **ssl.key.password**: SSL password.
+- **ssl.keystore.location**: SSL keystore location.
+- **ssl.keystore.password**: SSL keystore password.
+- **ssl.truststore.location**: SSL trust store location.
+- **ssl.truststore.password**: SSL trust store password.
+- **client.id**: Kafka producer client ID.
+- **security.providers**: establishes the security providers. The default value is _blank_.
+- **ssl.enabled.protocols**: SSL enabled protocols. Valid values are: TLSv1.2/TLSv1.3. The default value is _TLSv1.2,TLSv1.3_.
+- **ssl.endpoint.identification.algorithm**: SSL endpoint identification algorithm. You can leave the default value if you don't want to set it. The default value is _https_.
+- **ssl.keymanager.algorithm**: the algorithm used by the key manager factory for SSL connections. The default value is _SunX509_.
+- **ssl.keystore.type**: type of the repository of security certificates.
+- **ssl.provider**: the name of the security provider used for SSL connections.
+- **ssl.protocol**: the SSL protocol used to generate the SSLContext.
+- **timeout.millis**: the maximum time trying to poll before considering the current partitions as empty.
+- **max.poll.interval.ms**: the maximum delay between invocations of poll(). If poll() is not called before expiration of this timeout, the consumer is considered failed and the group will rebalance. This value should be less than _timeout.millis_.
+- **group.id**: specifies the name of the consumer group the consumers will belong to.
+
+>KLoadGen Consumer Sampler is compatible with **JSON** and **AVRO** messages. If **JSON** is being used, no extra configuration is needed, the sampler will work with the default deserialization settings. If **AVRO** is being used, the deserializer settings need to be configured in order to define how messages will be deserialized. You can choose whether you define the settings manually or whether you load a file with them.
```

Comment:
Here you have to replace of simple JSON with JSON Schema. Also don´t forget to add PROTOBUF as well, please.